src/staging/src/pages/parks.js

```diff
@@ -23,13 +23,13 @@ const ParksPage = ({ data }) => (
             <Grid item xs={12} key={document.node.id}>
               <p>
                 <a
-                  href={`/${
-                    document.node.slug
-                      ? document.node.slug
-                      : document.node.protectedAreaName
-                          .toLowerCase()
-                          .replace(/ /g, "-")
-                  }`}
+                  href={`/${(document.node.slug
+                    ? document.node.slug
+                    : document.node.protectedAreaName
+                      .toLowerCase()
+                      .replace(/ /g, "-"))
+                    .replace(/\/$|$/, `/`)
+                    }`}
```

Comment:
Nice timing, I discovered this on Friday while doing the SEO check, and was going to tackle it.

Reply:
It should still be tackled on the search results page.